vocab_files/attribute_concepts/targeted-survey-type.ttl

```diff
@@ -9,7 +9,7 @@ PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
     dcterms:source "McCallum K, Bignall J, Laws M, O’Neill S, Sparrow B. (unpublished draft) ‘Targeted Surveys module’ in ….. (eds) Ecological Field Monitoring Protocols Manual: Standardising environmental monitoring and data systems for improved decision making. Draft v 0.1 Report to DAWE. TERN, Adelaide." ;
     rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
     skos:definition "The type of targeted survey undertaken." ;
-    skos:prefLabel "survey type" ;
+    skos:prefLabel "targeted survey type" ;
     schema:url "https://github.com/ternaustralia/dawe-rlp-vocabs/tree/master/vocab_files/attribute_concepts/survey-type.ttl" ;
```

Comment:
This file was renamed which means the value of `schema:url` needs to change as well.

```suggestion
    schema:url "https://github.com/ternaustralia/dawe-rlp-vocabs/tree/master/vocab_files/attribute_concepts/targeted-survey-type.ttl" ;
```